cleverhans/tf2/attacks/fast_gradient_method.py

```diff
@@ -0,0 +1,65 @@
+"""The Fast Gradient Method attack."""
```

Comment:
Is this a new implementation or was it simply moved?

Reply:
This was simply moved.